godtools/Managers/TractManager.swift

```diff
@@ -16,57 +16,64 @@ class TractManager: NSObject {
 
 extension TractManager {
     
-    func loadResource(resource: String) -> (pages: [XMLIndexer], colors: TractColors) {
-        let manifestPath = resource + "-manifest"
-        let xmlData = loadXMLFile(manifestPath)
-        let manifest = xmlData["manifest"]
-        
-        let primaryColorString: String = (manifest.element?.attribute(by: "primary-color")?.text)!
-        let primaryTextColorString: String = (manifest.element?.attribute(by: "primary-text-color")?.text)!
-        let textColorString: String = (manifest.element?.attribute(by: "text-color")?.text)!
-        
+    func loadResource(resource: DownloadedResource, language: Language) -> (pages: [XMLIndexer], colors: TractColors) {
         var pages = [XMLIndexer]()
-        for child in manifest.children {
-            if child.element?.name == "page" {
-                let page = loadPage(child)
-                pages.append(page)
+        let tractColors = TractColors()
+        
+        let translation = resource.getTranslationForLanguage(language)
+        if translation != nil {
+            let manifestPath = translation?.manifestFilename
+            if manifestPath != nil {
+                let xmlData = loadXMLFile(manifestPath!)
+                let manifest = xmlData?["manifest"]
+                
+                if manifest != nil {
+                    let primaryColorString: String = (manifest!.element?.attribute(by: "primary-color")?.text) ?? "rgba(59, 164, 219, 1)"
```

Comment:
Also, lets create a separate class or struct to store the app default colors and reference them from here.